src/ProfileEndpoints.jl

```diff
@@ -246,11 +271,39 @@ end
 
 end  # if isdefined
 
+###
+### Debug super-endpoint
+###
+
+function debug_profile_endpoint_with_stage_path(stage_path = nothing)
+    stage_path = stage_path === nothing ? tempdir() : stage_path
```

Comment:
put this into the default value instead? `(stage_path = tempdir())`?

Reply:
I'm not a big of directly inling function calls on parameters as you are suggesting. The fact that they are separate here is intentional and a preference of mine.